src/components/Search.js

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import { SearchButton } from './SearchButton';
+import { CustomersTable } from './custTable.js';
```

Comment:
The convention is to name component files exactly the same as the component name, so in this case `CustomerTable`. And the `.js` is unnecessary here - it'll assume you meant `.js` if you leave it off

Reply:
Also, generally the convention is to use `export default` on your components. So then here you would import like `import SearchButton from ...`